Sources/Rendering/Core/Picker/index.d.ts

```diff
@@ -21,39 +21,34 @@ export interface vtkPicker extends vtkAbstractPicker {
 	getActors(): vtkActor[];
 
 	/**
-	 * Get a pointer to the dataset that was picked (if any).
+	 * Get the dataset that was picked (if any).
 	 */
 	getDataSet(): any;
 
 	/**
 	 * Get mapper that was picked (if any)
 	 */
-	getMapper(): null | vtkMapper;
+	getMapper(): Nullable<vtkMapper>;
 
 	/**
 	 * Get position in mapper (i.e., non-transformed) coordinates of pick point.
 	 */
-	getMapperPosition(): number[];
+	getMapperPosition(): Vector3;
 
 	/**
-	 * 
+	 * Get a list of the points the actors returned by getActors were intersected at.
 	 */
-	getMapperPositionByReference(): number[];
-
-	/**
-	 * Get a list of the points the actors returned by GetProp3Ds were intersected at.
-	 */
-	getPickedPositions(): number[];
+	getPickedPositions(): Vector3[];
 
 	/**
 	 * Get tolerance for performing pick operation.
 	 */
 	getTolerance(): number;
 
 	/**
-	 * FIXME: this needs to be check again
+	 * Invoke a pick change event.
```

Comment:
Why remove the FIXME ? 
If there is no need to be "check again", please explain in the commit message.

Reply:
the FIXME was set to check what the methods returns exactly